form/bootstrap4.rst

```diff
@@ -0,0 +1,116 @@
+======================
+
+Symfony provides several ways of integrating Bootstrap into your application. The
+most straightforward way is to just add the required ``<link>`` and ``<script>``
+elements in your templates (usually you only include them in the main layout
+template which other templates extend from):
+
+.. code-block:: html+twig
+
+    {# templates/base.html.twig #}
+
+    {# beware that the blocks in your template may be named different #}
+    {% block head_css %}
+        <!-- Copy CSS from https://getbootstrap.com/docs/4.0/getting-started/introduction/#css -->
+    {% endblock %}
+    {% block head_js %}
+        <!-- Copy JavaScript from https://getbootstrap.com/docs/4.0/getting-started/introduction/#js -->
+    {% endblock head %}
+
+If your application uses modern front-end practices, it's better to use
+:doc:`Webpack Encore </frontend>` and follow :doc:`this tutorial </frontend/encore/bootstrap>`
+to import Bootstrap's sources into your SCSS and JavaScript files.
+
+The next step is to configure the Symfony application to use Bootstrap 4 styles
+when rendering forms. If you want to apply them to all forms, define this
+configuration:
+
+.. configuration-block::
+
+    .. code-block:: yaml
+
+        # config/packages/twig.yaml
+        twig:
+            form_themes: ['bootstrap_4_layout.html.twig']
+
+    .. code-block:: xml
+
+        <!-- config/packages/twig.xml -->
+        <?xml version="1.0" encoding="UTF-8" ?>
+        <container xmlns="http://symfony.com/schema/dic/services"
+            xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+            xmlns:twig="http://symfony.com/schema/dic/twig"
+            xsi:schemaLocation="http://symfony.com/schema/dic/services
+                http://symfony.com/schema/dic/services/services-1.0.xsd
+                http://symfony.com/schema/dic/twig
+                http://symfony.com/schema/dic/twig/twig-1.0.xsd">
+
+            <twig:config>
+                <twig:form-theme>bootstrap_4_layout.html.twig</twig:form-theme>
+                <!-- ... -->
+            </twig:config>
+        </container>
+
+    .. code-block:: php
+
+        // config/packages/twig.php
+        $container->loadFromExtension('twig', array(
+            'form_themes' => array(
+                'bootstrap_4_layout.html.twig',
+            ),
+
+            // ...
+        ));
+
+If you prefer to apply the Bootstrap styles on a form to form basis, include the
+``form_theme`` tag in the templates where those forms are used:
+
+.. code-block:: twig
+
+    {# ... #}
+    {# this tag only applies to the forms defined in this template #}
+    {% form_theme form 'bootstrap_4_layout.html.twig' %}
+
+    {% block body %}
+        <h1>User Sign Up:</h1>
+        {{ form(form) }}
+    {% endblock body %}
+
+Accessibility
+-------------
+
+The Bootstrap 4 framework has done a good job making it accessible for function
+variations like impaired vision and cognitive ability. Symfony has taken this one
```

Comment:
Just asking: does this term sounds good to you? -> `function variations`  Should it be `functional variations` or even `functional diversity`?

Reply:
Im not sure. It is the perfect word in Swedish. We may need a native English speaker. 

My intent is to not have the word "disabilities" or anything that could be connected to a negative tone.